### PR TITLE
Armstrong Numbers

### PR DESCRIPTION
**Pull request by tongkiat**
Added a new exercise (Armstrong Numbers).

Please review.

Thanks.

### CODE REVIEW

File: exercises/armstrong-numbers/example.scm [hunk 1]
```diff
@@ -0,0 +1,17 @@
+(import (rnrs))
+
+(define (list-digits n)
+  (let loop ([n n]
+             [ms '()])
+    (let-values ([(d m) (div-and-mod n 10)])
+      (if (zero? n)
```
Comment (jitwit) [inline exercises/armstrong-numbers/example.scm:7]: Not hugely important, but it probably makes sense to check `zero?` before calling `div-and-mod`

File: exercises/armstrong-numbers/test.scm [hunk 2]
```diff
@@ -0,0 +1,91 @@
+(load "armstrong-numbers.scm")
+
+(use-modules (srfi srfi-64))
+
+(test-begin "armstrong-numbers")
+
+(test-eq "Zero is an Armstrong number"
+  (armstrong-number? 0)
+  #t)
```
Comment (jitwit) [inline exercises/armstrong-numbers/test.scm:9]: Since these are returning booleans, maybe `test-assert`? https://srfi.schemers.org/srfi-64/srfi-64.html

**Comment by jitwit** [comment]
This looks good. You probably also want to add this to `config.ss` and `config.json` as with the other problem so it goes live on the site

File: exercises/armstrong-numbers/test.scm (hunk 2 above)
Comment (tongkiat) [reply exercises/armstrong-numbers/test.scm:9]: Hmm..., I thought it'd be clearer to the students that we're expecting `#t` or `#f`.  Using `(test-assert ...)` may obfuscate that.  Anyway, I'll change it.  Thanks.

Comment (jitwit) [reply exercises/armstrong-numbers/test.scm:9]: Your choice, I think your reasoning makes sense as well

**Comment by tongkiat** [comment]
Comments?

**Comment by jitwit** [comment]
Looks good!

**Pull request merged**